tests/foreman/api/test_registration.py

```diff
@@ -107,12 +107,22 @@ def test_positive_allow_reregistration_when_dmi_uuid_changed(
     """
     uuid_1 = str(uuid.uuid1())
     uuid_2 = str(uuid.uuid4())
-    rhel_contenthost.install_katello_ca(target_sat)
     target_sat.execute(f'echo \'{{"dmi.system.uuid": "{uuid_1}"}}\' > /etc/rhsm/facts/uuid.facts')
-    result = rhel_contenthost.register_contenthost(module_org.label, lce=ENVIRONMENT)
-    assert result.status == 0
-    result = rhel_contenthost.execute('subscription-manager clean')
+    command = target_sat.api.RegistrationCommand(
+        organization=module_org,
+        activation_keys=[module_ak_with_synced_repo.name],
+        location=module_location,
+    ).create()
+    result = rhel_contenthost.execute(command)
     assert result.status == 0
+    rhel_contenthost.unregister()
     target_sat.execute(f'echo \'{{"dmi.system.uuid": "{uuid_2}"}}\' > /etc/rhsm/facts/uuid.facts')
-    result = rhel_contenthost.register_contenthost(module_org.label, lce=ENVIRONMENT)
+    command = target_sat.api.RegistrationCommand(
+        organization=module_org,
+        activation_keys=[module_ak_with_synced_repo.name],
+        location=module_location,
+        force=True,
+    ).create()
+    rhel_contenthost.execute(command)
```

Comment:
```suggestion
```

Reply:
nice catch, it is duplicated line

Reply:
This was a workaround for the issue but we need to skip the test case until this is fixed. I will add the skip marker.